src/config.rs

```diff
@@ -21,6 +21,7 @@ pub struct SymbolicTarget {
     pub owner: Option<UnixUser>,
     #[serde(rename = "if")]
     pub condition: Option<String>,
+    pub recursive: Option<bool>,
```

Comment:
This should probably be a `bool`, and by default deserialize to `true` when it's missing. Look into `serde field attributes` to make that happen